src/sentry/api/paginator.py

```diff
@@ -556,6 +559,8 @@ def get_item_key(self, item, for_prev=False):
             value_type = type(value)
             if value_type is float:
                 return math.floor(value) if self._is_asc(for_prev) else math.ceil(value)
+            elif value_type is str:
+                return value.lower()
```

Comment:
Actually wondering if `case_insensitive_sort` should just be a value passed straight to the `CombinedQuerysetPaginator`, so that we can check it here and only lower if we're using case insensitive pagination.

Might make sense for it to be renamed to `case_insensitive` since it covers both pagination and the sort.